e2e_tests/helpers.py

```diff
@@ -246,4 +246,4 @@ async def get_workspace_owner_token(admin_token, workspace_id, verify) -> Option
 
         token = responseJson["access_token"]
 
-        return token if (response.status_code == status.HTTP_200_OK) else None
+        return token, scope_uri
```

Comment:
What should happen now if we get a response which is not 200?

Reply:
An exception is thrown a few lines above if there is no access token in the response. It would have never met a false condition `response.status_code == status.HTTP_200_OK`.